.github/workflows/build-test-prValidation.yml

```diff
@@ -44,10 +44,10 @@ jobs:
           # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
           fetch-depth: 0
 
-      - name: Set up JDK 8
+      - name: Set up JDK 11
         uses: olafurpg/setup-scala@v10
         with:
-          java-version: adopt@1.8.0
+          java-version: adopt@1.11
```

Comment:
@johanandren seems like the latest Paradox doesn't work with JDK 8, so I switched to 11. We have builds with 8 in the nightlies. If that becomes a problem we could add a compile with 8 job for pr validation.